ui/apps/platform/src/Containers/Compliance/List/tableColumns.js

```diff
@@ -179,10 +233,11 @@ export function getColumnsByEntity(entityType, standards) {
         case resourceTypes.DEPLOYMENT:
             return getDeploymentColumns(filteredStandards);
         default:
-            return controlColumns;
+            return [];
```

Comment:
Nice - removing the implicit last case and explicitly handling it at the call site.